src/style.css

```diff
@@ -0,0 +1,93 @@
+@import "./stars.css";
+
+* {
```

Comment:
don`t set styles for *, set it only where it needed. You can create additional class like 'margin-reset' and use it in your html if there are a lot of cases